cloudify_aws/lambda_serverless/resources/function.py

```diff
@@ -83,6 +83,8 @@ def invoke(self, params):
         '''
         params = params or dict()
         params.update(dict(FunctionName=self.resource_id))
+        if params.get('Payload'):
+            params['Payload'] =  _encode_payload(params['Payload'])
```

Comment:
@kfrukacz-gs sounds good, to make this also work with file-like object we should also add support for that